[2025-05-21T10:10:47.491+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-21T10:10:47.544+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: merge_sources_pipeline.preprocessing manual__2025-05-21T10:10:27.021421+00:00 [queued]>
[2025-05-21T10:10:47.567+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: merge_sources_pipeline.preprocessing manual__2025-05-21T10:10:27.021421+00:00 [queued]>
[2025-05-21T10:10:47.567+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-05-21T10:10:47.595+0000] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): preprocessing> on 2025-05-21 10:10:27.021421+00:00
[2025-05-21T10:10:47.614+0000] {standard_task_runner.py:72} INFO - Started process 1616 to run task
[2025-05-21T10:10:47.622+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'merge_sources_pipeline', 'preprocessing', 'manual__2025-05-21T10:10:27.021421+00:00', '--job-id', '125', '--raw', '--subdir', 'DAGS_FOLDER/inference_dag.py', '--cfg-path', '/tmp/tmpr4nuwoq7']
[2025-05-21T10:10:47.629+0000] {standard_task_runner.py:105} INFO - Job 125: Subtask preprocessing
[2025-05-21T10:10:47.732+0000] {task_command.py:467} INFO - Running <TaskInstance: merge_sources_pipeline.preprocessing manual__2025-05-21T10:10:27.021421+00:00 [running]> on host c735ccec6410
[2025-05-21T10:10:47.906+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='merge_sources_pipeline' AIRFLOW_CTX_TASK_ID='preprocessing' AIRFLOW_CTX_EXECUTION_DATE='2025-05-21T10:10:27.021421+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-21T10:10:27.021421+00:00'
[2025-05-21T10:10:47.910+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-05-21T10:10:47.947+0000] {warnings.py:109} WARNING - /opt/***/scripts/preprocess.py:41: UserWarning: Could not infer format, so each element will be parsed individually, falling back to `dateutil`. To ensure parsing is consistent and as-expected, please specify a format.
  df["Take-off Time (UTC)"] = pd.to_datetime(df["Take-off Time (UTC)"], errors='coerce')

[2025-05-21T10:10:47.974+0000] {python.py:240} INFO - Done. Returned value was: [{'Aircraft': 3, 'Engine': 5, 'Seats': 180.0, 'Origin Airport': 31, 'Destination Airport': 54, 'Distance Flown (km)': 5500, 'CO2 (kgCO2e)': 1287.93, 'Month': 1, 'Day': 9, 'Weekday': 3, 'Takeoff_Hour': 15}]
[2025-05-21T10:10:48.022+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-05-21T10:10:48.023+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=merge_sources_pipeline, task_id=preprocessing, run_id=manual__2025-05-21T10:10:27.021421+00:00, execution_date=20250521T101027, start_date=20250521T101047, end_date=20250521T101048
[2025-05-21T10:10:48.118+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-21T10:10:48.180+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-21T10:10:48.184+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
